lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py

```diff
@@ -105,6 +106,11 @@ def _query_state_of_services(self, service_status):
             print(f'  {active_state}')
             print(f'  {start_time}')
 
+    def get_failed_service_info(self, failed_service):
```

Comment:
Currently this tests only runs for a systemd setup. But if TBD (PR from @mpfirrmann) gets merged in its current state then calling this function does not really give a helpful output. I assume the `errors='ignore'` will just cause an output of `Unit eventd.service could not be found.`.